openapi.yaml

```diff
@@ -1,279 +1,678 @@
+      description: 'Number of results to return per page. Used in pagination. Default value is 10.'
+      required: false
+      schema:
         type: string
-        format: uri
-        description: The last page of data
-      prev:
+        maximum: 500
+        default: 25
+        minimum: 1
+    pageNumber:
+      in: query
+      name: page[number]
+      description: 'Page number of results. Used to paginate through results.'
+      required: false
+      schema:
         type: string
-        format: uri
-        description: The previous page of data
-      next:
-        type: string
-        format: uri
-        description: The next page of data
-  PetResource:
-    properties:
-      id:
-        type: string
-        description: A unique ID of pet
-      type:
-        $ref: '#/definitions/PetType'
-      links:
-        $ref: '#/definitions/SelfLink'
-      attributes:
-        properties:
-          name:
+        default: 1
+        minimum: 1
+    isOpen:
+      in: query
+      name: filter[isOpen]
+      description: Set true to search only for results that are open right now
+      required: false
+      schema:
+        type: boolean
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://api.oregonstate.edu/oauth2/token
+          scopes:
+            full: Full access to the API
+  responses:
+    '400':
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '404':
+      description: Not found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '409':
+      description: Conflict
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '500':
+      description: Internal server error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+  schemas:
+    LocationResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          $ref: '#/components/schemas/LocationsResourceObject'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesResourceObject'
+    LocationSetResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationsResourceObject'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesIncluded'
+    ServicesResultObject:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          $ref: '#/components/schemas/ServicesResourceObject'
+    ServicesIncluded:
+      properties:
+        id:
+          type: string
```

Comment:
Can you add description and example for all of these fields? Example: https://github.com/osu-mist/persons-api/blob/v2/openapi.yaml#L846-L849